contents/swift/welcome-to-swift.md

```diff
@@ -12,44 +12,47 @@
 
 # Swift
 
-Authors: [Ch'ng Ming Shin](https://github.com/ablyx/cs3281-website/blob/mingshin-week6-progress/students/AY1617S2/ch'ngMingShin/Ch'ngMingShin-Resume.md), [Yong Zhi Yuan](https://github.com/Zhiyuan-Amos)
+Authors: [Ch'ng Ming Shin](https://github.com/ablyx/cs3281-website/blob/mingshin-week6-progress/students/AY1617S2/ch'ngMingShin/Ch'ngMingShin-Resume.md), [Yong Zhi Yuan](https://github.com/Zhiyuan-Amos), [Jiang Chunhui](https://github.com/Adoby7)
 
-# Overview
+## What is Swift?
 
-Welcome to Swift, THE preferred language to do iOS programming. Introduced in 2014, Swift is a fast, modern and safe programming language which supports playgrounds, an innovative feature that allows programmers to experiment with Swift code and see the results immediately, without the overhead of building and running an app.
+Swift is a fast, modern and safe programming language mainly used for iOS programming.
+Introduced in 2014, Swift has concise and expressive syntax, especially when compared with its predecessor language **Objective-C**. On top of that, Swift also supports playgrounds, a feature that allows programmers to experiment with Swift code and see the results immediately (without the overhead of building and running an app).
 
-# Getting Started
+## Why should I learn Swift?
 
-Reading the [Language Guide in the official documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is definitely recommended, since it explains everything quite clearly, albeit quite verbose.
+1. **Learning Swift can help you land your dream career.**
+Swift is mainly used for iOS development and iOS developers are well paid. The [salary statistics](https://www.indeed.com/salaries/iOS-Developer-Salaries) shows that iOS developer in US can earn more than $120K per year. In addition, mobile application development is also becoming more popular as of recent times. A [survey in StackOverFlow](https://insights.stackoverflow.com/survey/2017#developer-profile-developer-type) shows that mobile development, although lesser than web development, still takes up a large share of the market.
 
-If you're just looking to take a dip into Swift, here are the [basics](https://guides.codepath.com/ios/Swift-Basics) you'll need to understand.
+1. **Swift is easy to learn.** Swift is designed to be a simple and understandable language by Apple. Its syntax resembles English, and thus is easy to read and understand. Any programmer who is familiar with Python, Java or C++ can learn Swift quickly! Even if you are not familiar with any language, Swift is an excellent language to start out with.
```

Comment:
The first two lines could use a rephrase. You are repeating the easy to understand point twice. 

Shall we combine it into one? "Swift is designed to be a simple and user-friendly language by Apple. Its syntax resembles English, making it easy to read and understand.